wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+---
+tags:
+  - omLN
+  - o!mLN
+  - o!mLN4
+  - o!mLN 4
+---
+
+# osu!mania LN Tournament 4
+
+![o!mLN4 logo](img/logo.png) 
+
+The **osu!mania LN Tournament 4** (***o!mLN4***) is a worldwide double-elimination 1v1 osu!mania 4-key tournament hosted by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600) and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061).  The tournament featured a variety of maps with a vast amount of sliders (or Long Notes) in its mappools. It is the 4th instalment of the osu!mania LN tournament.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2025-07-21/2025-08-18 |
+| Screening phase | 2025-08-19/2025-09-06 |
+| Qualifier Showcase | 2025-09-07 (14:00 UTC) |
+| Qualifiers | 2025-09-08/2025-09-14 |
+| Round of 64 | 2025-09-15/2025-09-21 |
+| Round of 32 | 2025-09-22/2025-09-28 |
+| Round of 16 | 2025-09-29/2025-10-05 |
+| Quarterfinals | 2025-10-06/2025-10-16 |
+| Semifinals | 2025-10-13/2025-10-19 |
+| Finals | 2025-10-20/2025-10-26 |
+| Grand Finals | 2025-10-27/2025-11-02 |
+
+## Prizes
+
+Prize pool are divided to two parts: one is Regular Prize Pool, allocated to tournament winner; the other one is OII Prize Pool, allocated to event winner.
```

Comment:
A bit wack to read, miscellaneous changes. Using future tense as this action has not happened yet, although using a present tense should be just as fine. Semantics.

```suggestion
The prize pool will be divided in two parts: one is the **Regular Prize Pool**, allocated to the tournament winner; the other one is the **OII Prize Pool**, allocated to the event winner.
```

Also, in my opinion, I believe we should explain what "OII" means. It was explained in the news post but there isn't any mention of it prior to the prize pool section. A user stumbling upon this will have no idea what that means.